did/util.go

```diff
@@ -255,31 +255,31 @@ func DecodeMultibasePublicKeyWithType(data []byte) ([]byte, cryptosuite.LDKeyTyp
 }
 
 // ConstructJWKVerificationMethod builds a DID verification method with a known LD key type as a JWK
-func ConstructJWKVerificationMethod(id, keyReference string, pubKey []byte, keyType cryptosuite.LDKeyType,
-	cryptoKeyType crypto.KeyType) (*VerificationMethod, error) {
-	if keyType != cryptosuite.JSONWebKey2020Type {
-		return &VerificationMethod{
-			ID:              keyReference,
-			Type:            keyType,
-			Controller:      id,
-			PublicKeyBase58: base58.Encode(pubKey),
-		}, nil
-	}
-
-	cryptoPubKey, err := crypto.BytesToPubKey(pubKey, cryptoKeyType)
+func ConstructJWKVerificationMethod(id, controller string, pubKey []byte, cryptoKeyType crypto.KeyType) (*VerificationMethod, error) {
+	pubKeyBytes, err := crypto.BytesToPubKey(pubKey, cryptoKeyType)
```

Comment:
Why is this names pubKeyBytes? Isn't this typed?

Reply:
good catch, updating